### PR TITLE
feat: Add reviews commands to the bot

### PR DESCRIPTION
**Pull request by mimotej**
# Related issues
Resolves: https://github.com/OqixDevs/OqixTS/issues/85

# Type of pull request
- [x] New feature
- [ ] Bug fix
- [ ] Refactoring
- [ ] Feature update
- [ ] Other

# Describe your PR in detail
This feature adds option to review courses with option to rate these reviews.
Adding reviews:
Kooha-2023-12-14-20-08-52
Displaying and rating reviews:
Kooha-2023-12-14-20-11-18

### CODE REVIEW

File: src/commands/review.ts [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+                },
+                subjectCode: {
+                    equals: subjectCode,
+                },
+            },
+        });
+        console.log(existingReview);
+        if (existingReview.length != 0) {
+            return interaction.editReply({
+                content: `You have already submitted review fro subject ${subjectCode}.`,
```
Comment (xvanick1) [inline src/commands/review.ts:76]: typo 'fro'

**Review by xvanick1 — CHANGES_REQUESTED** [review]
Looks, good. Found just a small typo.

File: src/commands/review.ts (hunk 1 above)
Comment (mimotej) [reply src/commands/review.ts]: Resolved

**Review by xvanick1 — APPROVED** [review]
/lgtm

**Pull request merged**